### ISSUE TITLE
Send Client Report When Fully Rate Limited

### ISSUE THREAD

**Issue by philipphofmann** [getsentry/sentry-java#3815]
### Description

See: https://github.com/getsentry/sentry-cocoa/issues/4468

**Comment by adinauer** [comment]
Is there a risk of the SDK flooding relay with client reports when relay would prefer the SDK to send nothing instead?

**Comment by philipphofmann** [comment]
Yes, there is. I changed the description to more than x discarded events and not more often than every x seconds. What about 10 discarded events and maximum every 30 seconds? That should be acceptable for Relay I guess.

**Comment by markushi** [comment]
Let's bump the priority in case we see there's a critical issue around this.
